### ISSUE TITLE
Rescale the score from 140 to 100?

### ISSUE THREAD

**Issue by vprelovac** [WebKit/Speedometer#401]
Current scale is inherted from Speedometer 2. Where browser went up to 500 in score.

Now max score I've seen is about 35. It makes sense to change the scale and make the max lower, maybe 100, or even 50.

**Comment by rniwa** [comment]
I don't think we want to do this for 3.0. For 4.0, it's too early to say one way or another.

**Comment by camillobruni** [comment]
I've got feedback from several people that they'd wish just a steady ever-increasing score between releases. While a bit arbitrary, it makes talking about the score quite a bit more intuititve.

**Comment by rniwa** [comment]
> I've got feedback from several people that they'd wish just a steady ever-increasing score between releases. While a bit arbitrary, it makes talking about the score quite a bit more intuititve.

What do you mean by "more intuitive"?

**Comment by camillobruni** [comment]
Comparing devices with a score of let's say 300 meant roughly the same thing with v2.1. With the v3.0 release, all of a sudden the printed score was 10x "slower" and thus purely from a publicity standpoint, the device might be perceived slower. 

Also with folks who are not dealing with speedometer on a frequent basis we see that there is an implicit assumption of where we are, and how much progress we made based of the score. If we drastically change the score between versions this vague notion is lost.

Agreed, this is all a bit fuzzy, and with new releases we're technically not providing the same benchmark. So, comparing scores between versions is definitely not scientifically correct.

**Comment by vprelovac** [comment]
As someone who originall started this post, I agree with the previous poster. The disruption has already been made and the scores 'reset' going from v2 to v3. I do not think that needs to be addressed any more.

What I think should be done is to make current scoring future proff so that another 'reset' is not needed again. People want 'stability' and predicatability. One thing I would suggest calibrating a score of one to 'reference machine/browser'.

This can be anything - pick a commonly used piece of hardware and the browser ti ships for. Something people would be able to relate to.

So then a score of 100 would mean 100x faster than reference. Will help tell good stories.
